generic_config_updater/gu_common.py

```diff
@@ -54,8 +54,7 @@ def get_config_db_as_json(self):
         return json.loads(text)
 
     def _get_config_db_as_text(self):
-        # TODO: Getting configs from CLI is very slow, need to get it from sonic-cffgen directly
-        cmd = "show runningconfiguration all"
+        cmd = "sonic-cfggen -d --print-data"
```

Comment:
>cmd = "sonic-cfggen -d --print-data" [](http://example.com/codeflow?start=8&length=36)

This is a valid performance improvement, and not related to "LaneReplacement Generator".

Could you extract it into a separate PR? also add some performance measurement result.

Reply:
I've made it a separate PR but got the coverage check failure.
That is because, in our test, the function is being mocked. If we do performance measurement, I think we need to put that in config CLI test.

Local manual test for comparison.
```
admin@vlab-01:~/time$ python check.py  | grep "\-\-\-"
show run all --- 0.7292575836181641 seconds ---
sonic-cfggen --- 0.0 seconds ---
```

Reply:
Offline discussed, we need to create an issue that "show runningconfiguration all" is slow, and we need to fix in its own implementation.

Reply:
My previous time check is wrong. Please check the below metric.

```
--- SINGLE RUN
show SINGLE --- 0.7178833484649658 seconds ---
sonic-cfggen SINGLE --- 0.3109145164489746 seconds ---
--- 10 RUNS
show AVERAGE --- 0.7603521585464478 seconds ---
sonic-cfggen AVERAGE --- 0.28761484622955324 seconds ---
```